modules/social_features/social_group/social_group.install

```diff
@@ -696,3 +699,78 @@ function social_group_update_10302() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Import flag "Mute Group Notifications" configurations.
+ */
+function social_group_update_10303() {
+  $configs = [
+    'install' => [
```

Comment:
We can never use install folders in an update hook, always use a static representation of this. 
e.g. an array or a `config/static`

Reply:
So, what is the best way to add new configurations now? By using update helper?

Reply:
I've changed it. Check it, please.